rpc/cluster/ReplicaInfo.java

```diff
@@ -62,7 +62,7 @@ public static ReplicaInfo ofProto(DatabaseProto.Database.Replicas.Res res) {
     Replica preferredSecondaryReplica() {
         return replicas.values().stream()
                 .filter(Replica::isPreferredSecondary)
-                .max(Comparator.comparing(Replica::term))
```

Comment:
This comparison is simply unnecessary, the server only ever selects one preferred secondary replica.